game-core/src/test/java/games/strategy/engine/lobby/client/ui/GameListModelTest.java

```diff
@@ -0,0 +1,63 @@
+package games.strategy.engine.lobby.client.ui;
```

Comment:
This test is actually a simplified version of LobbyGameTableModelTest and is renamed.